### ISSUE TITLE
Information : ace virus false positive

### ISSUE THREAD

**Issue by havena** [acemod/ACE3#2874]
for information, i have report false positive related to ace :

In relation to submission [3881126].

Upon further analysis and investigation we have verified your submission and, as such, the detection(s) for the following file(s) will be removed from our products:

```
    3CDFC521C81F430EC7125F4D87F4437F - ace_fcs.dll
    7A8E5E73D06D5E5AC350ED559FC1B90D - ace_clipboard.dll
    80CEC9C137DC9126691DA264CF241C8D - ace_parse_imagepath.dll
    9B51E1774C969C70D90F75FCB4A658C6 - ace_medical.dll
    BFBD1A78019D53914148E5F2EAD36206 - ace_advanced_ballistics.dll
    E867FC4A8F630D4AA3BC797FBA10C7CC - ace_break_line.dll
```

Place to get latest definition and check associated sequence number:
Norton: http://us.norton.com/security_response/definitions.jsp
SEP: https://www.symantec.com/security_response/definitions.jsp

If detection persists, please contact our support at:
Norton: https://support.norton.com/sp/en/us/home/current/info
SEP: https://support.symantec.com/en_US/endpoint-protection.54619.html

Decisions made by Symantec are subject to change if alterations to the Software are made over time or as classification criteria and/or the policy employed by Symantec changes over time to address the evolving landscape.

If you are a software vendor and would like to upload your software for proactive whitelisting, please complete the following form: https://submit.symantec.com/whitelist/

For more information on best practices to reduce false positives:
http://www.symantec.com/content/en/us/enterprise/white_papers/b-to_increase_downloads-instill_trust_first_WP.en-us.pdf

Sincerely,
Symantec Security Response
http://securityresponse.symantec.com

**Comment by commy2** [comment]
>Upon further analysis and investigation we have verified your submission and, as such, the detection(s) for the following file(s) will be removed from our products:

Nice. Let's hope they fix it.

**Comment by SilentSpike** [comment]
:+1: That's Symantec and Kaspersky both contacted for false positives

**Comment by bux578** [comment]
@SilentSpike https://www.virustotal.com/en/file/ca94f89ce77c54b89973f33a917068248a40ced0ef555a14e2d408d593f32ae9/analysis/

**Comment by SilentSpike** [comment]
I've now also reported the false positive to BitDefender and McAfee

**Comment by PabstMirror** [comment]
I rescaned on virustotal and the detection ratio went from 11 down to 6/54, (bit defender is now clean)

**Comment by SilentSpike** [comment]
Reported to Ikarus and QuickHeal now too, a handy site for reporting false positives quickly:

http://www.techsupportalert.com/content/how-report-malware-or-false-positives-multiple-antivirus-vendors.htm#List_Of_All_Vendors

**Comment by TheGrandVizier** [comment]
Bullguard also flags as false positive, fyi.

**Comment by headedhorseman101** [comment]
I got one on eset nod

**Comment by AntaresTheSarcastic** [comment]
The issue has returned - same files.  Bitdefender is now flagging them as viruses again.

**Comment by commy2** [comment]
See: https://github.com/acemod/ACE3/issues/2874#issuecomment-160202764

**Comment by wagenbach** [comment]
My antivirus isn't flagging them as positive for viruses, but Battleye blocks them now. Figured I'd drop a note here.

**Comment by Glowbal** [comment]
I assume this is fixed by #3905
